test/fixtures/WidenImage.html

```diff
@@ -24,6 +24,11 @@
         </div>
       </div>
     </div>
+    <div style='position:absolute'>
```

Comment:
If this is written as `position: absolute`, which is valid css, this test fails

Reply:
@joewalsh Fixed!